Classes/Emogrifier.php

```diff
@@ -533,12 +522,10 @@ private function removeImportantAnnotationFromNodeInlineStyle(\DOMElement $node)
      */
     private function getAllNodesWithStyleAttribute(\DOMXPath $xPath)
     {
-        $nodesWithStyleAttribute = $xPath->query('//*[@style]');
-        if ($nodesWithStyleAttribute === false) {
-            throw new \RuntimeException(
-                'Possibly malformed DOMXPath query expression or invalid DOMXPath object.',
-                1508961431
-            );
+        try {
+            $nodesWithStyleAttribute = $xPath->query('//*[@style]');
```

Comment:
Maybe we can also drop some of the parts where exceptions are caught and then silently ignored for cases where this points to a programming error in Emogrifier, not to not-so-clean (or broken) HTML/CSS. What do you think?

Reply:
And if we do so, we could do this in a separate clean-up PR before this one so we keep things (more) separate.